<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>consumerpage</class>
 <widget class="QWidget" name="consumerpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="CaddOrder">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>点餐</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getTable">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>预定餐位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewOrder">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查看小票</string>
   </property>
  </widget>
  <widget class="QLabel" name="bar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>顾客系统</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
